app/models/list.rb

```diff
@@ -14,4 +14,8 @@ def incompleted_items
     items.reject { |item| item.done? }
   end
 
+  def has_items?
+    !items.empty?
+  end
```

Comment:
@vlasiak Another way to do this without inversion is to use `present?` See [the details](http://apidock.com/rails/Object/present%3F). Also, please make sure that it doesn't make any queries into DB when it is called.